Tests/SonarScanner.MSBuild.Shim.Test/ScannerEngineInputGeneratorTest.GenerateResult.cs

```diff
@@ -494,38 +494,39 @@ public void GenerateResult_AdditionalProperties()
             new("key1", "value1"),
             new("key.2", "value two"),
             new("key.3", " "),
-            // Sensitive data should not be written
+            // Sensitive data should not be written to the SQProperties File
+            // Sensitive should be passed to the scanner-engine
```

Comment:
I'd put these comments before their respective reader creation, to make it clear why they differ there